test/e2e/apps/statefulset.go

```diff
@@ -882,6 +889,89 @@ var _ = SIGDescribe("StatefulSet", func() {
 			appTester.run()
 		})
 	})
+
+	framework.KubeDescribe("V1beta2 StatefulSet Scale Subresource [Feature:StatefulSet]", func() {
```

Comment:
Ya, we can put under "Basic StatefulSet functionality". The reason i separate it out is due to it is a v1beta2 test. As confirmed by you that  a v1beta1 StatefulSet is viable due to server side conversion (internal version), there is no need anymore for separating out the test.